ios/Classes/PMNotificationManager.m

```diff
@@ -42,6 +42,13 @@ - (void)stopNotify {
     _notifying = NO;
 }
 
+-(void)detachFromEngine {
+    if (_notifying) {
+        [self stopNotify];
```

Comment:
I'm not sure whether we should stop notifying here.

Reply:
Well, actually the `FlutterPlugin detachFromEngine` is the cleanup procedure during the engine deallocation, which is usually an action performed during the exit of the app. So leaving the notification on but the method channel is not available I don't think is usable in any way since that method channel will never be available again.